src/js/utils/loader.js

```diff
@@ -0,0 +1,29 @@
+export function showLoader() {
```

Comment:
```suggestion
export function showLoader(container = false) {
container = container ? container : document.body
```

Reply:
adds an option to put another container in as a parameter, if not added then it will default to document.body (like it is now)